test/members_page_test.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+require_relative './test_helper'
+require_relative '../lib/members_page.rb'
+
+describe 'MembersPage' do
+  around { |test| VCR.use_cassette('Members Page', &test) }
+
+  let :members_page do
+    url = 'http://sitl.diputados.gob.mx/LXIII_leg/listado_diputados_gpnp.php?tipot=TOTAL'
+    MembersPage.new(response: Scraped::Request.new(url: url).response)
+  end
+
+  subject do
+    party = members_page.party_groupings.find do |party_group|
+      party_group.party_id == party_id
+    end
+    party.members.find { |mem| mem.id == id }
+  end
+
+  describe 'party groupings' do
+    subject do
```

Comment:
This is clobbering an outer `subject`. If these are two different types of tests, they should be separately nested.